types/orchestrator/index.d.ts

```diff
@@ -1,107 +1,141 @@
-// Type definitions for Orchestrator
+// Type definitions for Orchestrator 0.3
 // Project: https://github.com/orchestrator/orchestrator
-// Definitions by: Qubo <https://github.com/tkQubo>
+// Definitions by: Qubo <https://github.com/tkQubo>, TeamworkGuy2 <https://github.com/TeamworkGuy2>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node" />
+
+import * as events from "events";
+import * as stream from "stream";
 import * as Q from "q";
 
-declare type Strings = string|string[];
+type _Sequencify = (tasks: Array<{ dep: string[]; }>, names: string[]) => { sequence: string[]; missingTasks: string[]; recursiveDependencies: string[]; };
+
+type _runTask = (task: Orchestrator.TaskFunc, done: (err: any, meta: Orchestrator.Meta) => void) => void;
```

Comment:
Sorry, forgot about those, was using those for reference while I was build the definition. 
They are the definitions for the node module 'sequencify' and the orchestrator lib/runTask.js file:
- https://www.npmjs.com/package/sequencify
- https://github.com/robrich/orchestrator/blob/master/lib/runTask.js
I'll remove those two lines and commit shortly.